### PR TITLE
ListView tests

### PR DESCRIPTION
**Pull request by muselesscreator**
Add tests for ListView component.

Split out the TableControls for ease of mocking.

moved url designations to `data/services/lms/urls`

JIRA: [AU-331](https://openedx.atlassian.net/browse/AU-331)

### CODE REVIEW

File: src/containers/ListView/TableControls.test.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import TableControls from './TableControls';
+
+jest.mock('@edx/paragon', () => ({
+  DataTable: {
+    TableController: () => 'DataTable.TableController',
```
Comment (leangseu-edx) [inline src/containers/ListView/TableControls.test.jsx]: TableControlBar
```suggestion
    TableControlBar: () => 'DataTable.TableControlBar',
```

**Review by leangseu-edx — CHANGES_REQUESTED** [review]
Change the typo and update the snapshot

**Comment by muselesscreator** [comment]
> Change the typo and update the snapshot

good catch and thx.  done :-)

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍

**Pull request merged**